spec/install/gemfile_spec.rb

```diff
@@ -66,4 +66,15 @@
       expect(out).to match(/You passed :lib as an option for gem 'rack', but it is invalid/)
     end
   end
+
+  if RUBY_ENGINE == "jruby"
```

Comment:
this should simulate the ruby engine instead of being a conditional -- i.e. `before { simulate_ruby_engine "jruby" }`